ch.tigerjython.TigerJython2.yml

```diff
@@ -0,0 +1,39 @@
+id: ch.tigerjython.TigerJython2
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk
+finish-args:
+  - --env=PATH=/app/jre/bin:/app/bin:/usr/bin
+  # X11 + XShm access
+  - --share=ipc
+  # - --socket=fallback-x11
+  # Wayland access
+  - --socket=x11
+  # GPU acceleration if needed
+  - --device=dri
+  # Needs to talk to the network:
+  - --share=network
+  # Needs to save files locally
+  - --filesystem=xdg-documents
+command: tigerjython
+modules:
+  - name: openjdk
+    buildsystem: simple
+    build-commands:
+      - /usr/lib/sdk/openjdk/install.sh
+  - name: tigerjython2
+    buildsystem: simple
+    build-commands:
+      - sed -i "s|/usr/bin/java|/app/jre/bin/java|g" tigerjython
+      - sed -i "s|/opt/tigerjython|/app/tigerjython|g" tigerjython
+      - install -Dm644 Lib/* -t /app/tigerjython/Lib
+      - install -Dm644 TestSamples/*  -t /app/tigerjython/TestSamples
+      - install -m755 tigerjython2.jar /app/tigerjython/tigerjython2.jar
+      - install -Dm755 tigerjython -t /app/bin
+      - desktop-file-edit --set-icon=ch.tigerjython.TigerJython2 --set-key=Exec --set-value="tigerjython %f" tigerjython.desktop
```

Comment:
desktop file has to be installed as ch.tigerjython.TigerJython2.desktop

Reply:
Where do I have to place it? (and perhaps I should place the icon too?)

How do I have to name the icon?